edi_oca/models/edi_exchange_consumer_mixin.py

```diff
@@ -44,6 +44,12 @@ class EDIExchangeConsumerMixin(models.AbstractModel):
         default={},
     )
     edi_has_form_config = fields.Boolean(compute="_compute_edi_config")
+    # TODO: rename to `edi_disable_auto`
+    disable_edi_auto = fields.Boolean(
```

Comment:
This is the old TODO I mentioned a long time ago. 
You can now reduce the override of this field only to the states.
CC @LoisRForgeFlow for edi_purchase and other modules.

Reply:
@LoisRForgeFlow @etobella ping

Reply:
Sorry @simahawk, were you expecting some action from our side? I thought it was more like a "FYI"

Reply:
@LoisRForgeFlow yeah, just wanted an ack and maybe a review on the rest :wink: